tsdb/meta.go

```diff
@@ -408,7 +408,7 @@ func (m *Measurement) DropSeries(seriesID uint64) bool {
 		}
 	}
 
-	return true
+	return false
```

Comment:
is this return value used anywhere? Maybe it's better to update the method to not return anything if we're not actually using it.